.spi.yml

```diff
@@ -0,0 +1,4 @@
+version: 1
+builder:
+  configs:
+    - documentation_targets: [AWSLambdaRuntime]
```

Comment:
Do we need to include `AWSLambdaRuntimeCore`?

Reply:
yes that is good idea